index.js

```diff
@@ -7,12 +7,42 @@ function ask(questionText) {
   });
 }
 
-start();
+//The computer then guesses within the predetermined range
+let computerGuess = (max, min) => {
+  let guess = (min + max) / 2;
+  return Math.floor(guess);
+};
 
+start();
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  let playing = true;
+  let upperBound = 100
+  let lowerBound = 1
+  let currentGuess = computerGuess(upperBound, lowerBound);
+  console.log(
+    "Let's play a game where you (human) make up a number and I (computer) try to guess it."
+  );
+  //The user picks a secret number
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n"
+  );
+  console.log("You entered: " + secretNumber);
+  while (playing) {
+    let yesOrNo = await ask("is your number " + currentGuess + "? ");
+    if (yesOrNo.toLowerCase().includes("yes")) {
+      console.log("You're correct, that is my number!");
```

Comment:
Maybe have the computer say "**I'm** correct..." since it's the computer guessing, not the human